peps/pep-0730.rst

```diff
@@ -111,26 +116,15 @@ a Framework. This also means that the common assumption that a Python module can
 construct the location of a binary module by using the ``__file__`` attribute of
 the Python module no longer holds.
 
-As with macOS, compiling a dynamic library requires the use of the ``--undefined
-dynamic_lookup`` option to avoid linking libPython into every binary module.
-This option currently raises a deprecation warning when it is used. This warning
-*was* previously raised on macOS builds as well; however, responses from Apple
-staff suggests this was unintentional, and they `did not intend to break the
-CPython ecosystem by removing this option
-<https://developer.apple.com/forums/thread/719961>`__. It is difficult to judge
-whether iOS support would fall under the same umbrella.
-
-Distribution
-------------
-
-Adding iOS as a Tier 3 platform only requires adding support for compiling an
-iOS-compatible build from an unpatched CPython code checkout. It does not
-require production of officially distributed iOS artefacts for use by end-users.
-
-If/when iOS is updated to Tier 2 or 1 support, there should be a process for
-producing iOS distribution artefacts. This could be in the form of an "embedded
-distribution" analogous to the Windows embedded distribution, or as a Cocoapod
-or Swift Package that could be added to an Xcode project.
+As with macOS, compiling a binary module that is accessible from a
+statically-linked build of Python requires the use of the ``--undefined
+dynamic_lookup`` option to avoid linking ``libPython`` into every binary module.
+However, on iOS, this compiler flag raises a deprecation warning when it is
+used. This warning has been observed on macOS as well - however, responses from
+Apple staff suggests that they `do not intend to break the CPython ecosystem by
+removing this option <https://developer.apple.com/forums/thread/719961>`__. As
```

Comment:
The warning on macOS has a different wording which doesn't mention deprecation.

Also, replace the link with a more recent and complete statement from Apple.

```suggestion
used. A warning from this flag has been observed on macOS as well - however, responses from
Apple staff suggest that they `do not intend to break the CPython ecosystem by
removing this option <https://github.com/python/cpython/issues/97524#issuecomment-1458855301>`__. As
```